.gitignore

```diff
@@ -132,4 +132,9 @@ dmypy.json
 .terraform*
 *.zip
 
-secrets.txt
\ No newline at end of file
+secrets.txt
+_env.py
+venv/*
+
+*.csv
+*.txt
```

Comment:
can you please make the ignore of *.txt more specific. We need to develop txt files such as requirements.txt